### DISCUSSION TITLE
BSEED 2-way switch identified as TuYa 1 gang switch TS0001 - How to Pair?

### DISCUSSION THREAD

**Discussion by Piscator74** [Koenkk/zigbee2mqtt#15519] [Devices]
I recently got a [BSEED 2-way switch](https://www.aliexpress.com/item/1005003445267386.html), which is not listed as supported. Both switches get identified as a [TuYa 1 gang switch TS0001](https://www.zigbee2mqtt.io/devices/TS0001.html). So far, so good. 

The problem is that I cannot find a way to pair both switches (so that if you press 1 switch, the other turns on as well and vice versa). Anyone has experience with (other) 2 way configurations with this switch type and how to pair these? Of course, I can do that programmatically in a domotics client of zigbee2mqtt, but -if possible- a solution in zigbee2mqtt is better.

**Comment by olili** [comment]
see [Feature Request ](https://github.com/Koenkk/zigbee2mqtt/issues/13323)

Please support/vote for the feature request. 

O.
